x-pack/solutions/observability/plugins/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -35,7 +34,6 @@ import { RuleTypeModel } from '@kbn/triggers-actions-ui-plugin/public';
 import { useBreadcrumbs } from '@kbn/observability-shared-plugin/public';
 import dedent from 'dedent';
 import { AlertFieldsTable } from '@kbn/alerts-ui-shared/src/alert_fields_table';
-import { dashboardServiceProvider } from '@kbn/response-ops-rule-form/src/common';
```

Comment:
The dashboardServiceProvider was introduced as part of this [PR](https://github.com/elastic/kibana/pull/219019) to be used in 2 places:
- in the rule form of the response ops plugin
- in the o11y alert details page

I suggest we create another issue to replace the usage of dashboardServiceProvider with the new api in the response ops rule form as well. As part of the new issue we should completely remove the dashboard service provider and the dependency to contentManagement plugin that was introduced to fix a cycle dependency issue. I will file an issue for this.